src/styles/partials/components/_site-nav.scss

```diff
@@ -74,6 +74,7 @@ $mobile-menu-width: 245px;
           text-align: center;
           margin-right: $margin-small;
           width: 40px;
+          font-size: 30px;
```

Comment:
can you please create a variable for this in the _icon.scss variable file?

Reply:
done